Prettify markdown, fingerprint autoparsing

TheTechromancer opened the pull request:
Greetings,

This PR uses [prettier](https://prettier.io/) to automatically prettify the markdown table. This is intended to make it easier to read and maintain.

A github workflow has been added which automatically parses the markdown, generates `fingerprints.json`, and commits it to the repo.

![image](https://user-images.githubusercontent.com/20261699/206275630-8e87a0ba-c5fd-41fb-bb6f-996a33f56bc7.png)

EdOverflow commented on scripts/gen_fingerprints.py:
```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env python3
```
Thank you for these contributions, @TheTechromancer. Would it be possible to include a comment explaining of this purpose of this file for future maintainers? It's just so to make things easier for others when they want to contribute to the project.

TheTechromancer replied:
Sure! Note that right now, `fingerprints.json` doesn't have all the information needed for automated detection. There are still a few steps we want to take, including expanding the table to include other fields such as the vulnerable domains involved. But this PR is meant to lay the groundwork, as those steps will take significantly more time and testing.

TheTechromancer commented:
Update:
- Added a system for auto-verifying fingerprints
- Added colums `Verified` and `Domains` to assist with automated tooling
![image](https://user-images.githubusercontent.com/20261699/206793258-07ba4558-5e81-445c-9cc4-d32fc9e55bfa.png)

EdOverflow commented:
Still not too sure about the `Autocheck` column name. I will see if I can come up with a more descriptive column name.

pdelteil commented:
> This looks good to me. I plan on merging this after I have reviewed @pdelteil's PRs.

The PR's I sent are very simple, and it's been two weeks already.  😢

EdOverflow commented:
> Still not too sure about the `Autocheck` column name. I will see if I can come up with a more descriptive column name.

@TheTechromancer, would something along the lines of "Verified by CI/CD" be more suitable as a column name? We have to keep in mind that the README has to be presentable to humans.

TheTechromancer commented:
> @TheTechromancer, would something along the lines of "Verified by CI/CD" be more suitable as a column name? 

That sounds good to me. I'll make the change
